README.rst

```diff
@@ -216,3 +217,16 @@ Documentation for other versions
 * `v0.1.1 <https://www.pygmt.org/v0.1.1>`__
 * `v0.1.0 <https://www.pygmt.org/v0.1.0>`__
 * `v0.0.1a0 <https://www.pygmt.org/0.0.1a0>`__
+
+Compatibility with Python and GMT versions
+------------------------------------------
+
+======= ========== =========
+PyGMT   GMT        Python
+======= ========== =========
```

Comment:
We can add the 0.3.0 entry now or later.

Reply:
Let's add it later, not too sure yet when we want to cut a 0.3.0 release?

Reply:
I think we can release v0.3.0 two weeks before GMT v6.2.0.